client/src/components/Answers.jsx

```diff
@@ -91,6 +92,49 @@ class Answers extends React.Component {
       showAll: true
     })
   }
+  convertToMonthYear(date) {
+    var split = date.split('-')
```

Comment:
I bet you could use a switch statement here!

Reply:
or, you could strip the 0 off the front of the split, put the month names into an array (starting with a dummy value in the zero index), and then you could refer to each month by monthsArray[split] ... just an idea if you wanted to shorten this! (ofc the way it currently is works great too!)

Reply:
I should look into that...